UI/window-basic-main-profiles.cpp

```diff
@@ -783,6 +783,16 @@ void OBSBasic::ChangeProfile()
 	auth.reset();
 	DestroyPanelCookieManager();
 
+#ifdef YOUTUBE_ENABLED
+	OBSBasic* main = OBSBasic::Get();
```

Comment:
Place asterisk next to the variable name, not the type.

Reply:
Done.